### PR TITLE
Removetimertext

### PR DESCRIPTION
**Pull request by bpage-gamedev**
Remove the Scouting UI timer text and change the starting scene to Home.unity.

### CODE REVIEW

File: Assets/Scripts/UI/PickupPup/DogProfile/DogProfileButtonController.cs [hunk 1]
```diff
@@ -65,18 +65,20 @@ protected override void setReferences()
 
     #endregion
 
-	public void Init(DogProfile profile, DogDescriptor[] dogs)
+    public void Init(DogProfile profile, DogDescriptor[] dogs)
```
Comment (gracebarsny) [inline Assets/Scripts/UI/PickupPup/DogProfile/DogProfileButtonController.cs]: It looks like this file is filled with empty changes like this. Safer to remove this from your PR.

File: Assets/Scripts/UI/PickupPup/DogSlot/DogCollarSlot.cs [hunk 2]
```diff
@@ -121,12 +115,7 @@ protected override void callOnOccupiedSlotClick(Dog dog)
     #endregion
 
     #region UIElement Overrides 
-
-    public override void SetText(string text)
-    {
-        timerText.text = text;
-    }
-
+    //None at the moment...
```
Comment (gracebarsny) [inline Assets/Scripts/UI/PickupPup/DogSlot/DogCollarSlot.cs]: Delete this region if there are no longer any more UIElement Overrides.

File: Assets/Scripts/UI/PickupPup/DogProfile/DogProfileButtonController.cs (hunk 1 above)
Comment (bpage-gamedev) [inline Assets/Scripts/UI/PickupPup/DogProfile/DogProfileButtonController.cs]: I'll look for those later today. The radials work again. Something about removing the actual timer object from the dogCollarSlot items in the editor broke them.

**Comment by imann24** [comment]
@bpage-gamedev is this ready to be reviewed again or are you still looking into the prefab issues?

**Pull request merged**